ptu/global.ini

```diff
@@ -73595,4 +73595,5 @@ hud_darkwater_turnin_plat=Du hast eine F8C mit LTI erhalten. Starte deinen Clien
 hud_darkwater_turnin_gold=Du hast eine F8C für 1 Tag geliehen bekommen und kannst das Schiff kaufen.
 hud_State=Zustand
 vehicle_NameARGO_ATLS=Argo ATLS
+INTERIORMAP_common_transit_platform=Transport-Plattform
```

Comment:
Hast das zufällig auch im Spiel mal gegengecheckt? Ob das Problem behoben wurde? Falls nicht ist auch in Ordnung, mach ich dann morgen oder so.

Reply:
Nein habe ich nicht